src/content/docs/codestream/how-use-codestream/discuss-code.mdx

```diff
@@ -3,11 +3,12 @@ title: Discuss code on CodeStream
 metaDescription: "How to enable code discussions using New Relic CodeStream codemarks."
 redirects:
   - /docs/codestream/workflow-overview/discuss-code
+  - /docs/codestream/codestream-ui-overview/codemarks-section
 ---
 
 ## What is a codemark? [#codemark]
 
-Quite simply, a codemark is a discussion connected to the code. It could be a question, a suggestion, a bug report, or documentation. All of these discussions are saved, anchored to the blocks of code they refer to, so that they can be leveraged in the future. It could be a new developer joining the team, a developer trying to fix a bug in someone else’s code, or even just you trying to remember why you made that change six months ago. Whatever the case, CodeStream helps you understand the code by surfacing the discussions in a contextual way.
+Quite simply, a codemark is a [discussion](/docs/codestream/how-use-codestream/discuss-code) connected to the code. It could be a question, a suggestion, a bug report, or documentation. All of these discussions are saved, anchored to the blocks of code they refer to, so that they can be leveraged in the future. It could be a new developer joining the team, a developer trying to fix a bug in someone else’s code, or even just you trying to remember why you made that change six months ago. Whatever the case, CodeStream helps you understand the code by surfacing the discussions in a contextual way.
```

Comment:
Need to remove the "discussion" link, as it's self referential.

Reply:
Right, removed!